targets/libyaml/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd /src/libyaml
+
+./bootstrap
+./configure
+make "-j$(nproc)"
+
+$CXX $CXXFLAGS -std=c++11 -Iinclude \
+    /src/libyaml_fuzzer.cc -o /out/libyaml_fuzzer \
```

Comment:
Can you please use $SRC and $OUT. The format just changed, sorry for inconvienance.